bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -0,0 +1,115 @@
+# OpenTherm Gateway Binding
+
+This binding is used to integrate the OpenTherm Gateway into OpenHAB2. The OpenTherm Gateway is a module designed by Schelte Bron that is connected in between a boiler and a thermostat that communicates using the OpenTherm protocol.
+
+More information on the OpenTherm Gateway can be found at http://otgw.tclcode.com/
+
+This binding is still under development. Please visit https://github.com/ArjenKorevaar/openhab2-openthermgateway-binary to download the lastest compiled test version.
+
+Comments can be submitted to the topic on OpenHAB community at https://community.openhab.org/t/39160
+
+## Supported Things
+
+The OpenTherm Gateway binding currently only supports one thing, and that's the gateway itself.
+
+## Discovery
+
+The binding does not support auto discovery.
+
+## Binding Configuration
+
+The binding itself does not require any configuration.
+
+## Thing Configuration
+
+The binding is designed to support various ways of connecting to the OpenTherm Gateway, but currently only supports a TCP socket connection. The configuration settings for the thing are Hostname and Port, which are used to open the socket to the gateway.
+
+Future types of connections may require other settings, such as a COM port.
+
+
+
+
+
+
+## Channels
+The OpenTherm Gateway supports the following channels:
+- Room temperature
```

Comment:
Please add the name of the channels (in a table) and add what items can be connected to these channels.